superset/views/base_api.py

```diff
@@ -490,6 +490,12 @@ def related(self, column_name: str, **kwargs: Any) -> FlaskResponse:
 
         # handle pagination
         page, page_size = self._handle_page_args(args)
+
+        ids = args.get("include_ids")
+        if page and ids:
+            # pagination with forced ids is not supported
+            return self.response_400()
```

Comment:
Should we send a 422 instead? https://softwareengineering.stackexchange.com/questions/329229/should-i-return-an-http-400-bad-request-status-if-a-parameter-is-syntactically

Reply:
That's a good idea - will update 👍